### DISCUSSION TITLE
# using this is against DRY principle

### DISCUSSION THREAD

**Discussion by doganmeh** [djangopackages/djangopackages#653] [General]
It kind of bothers me seeing '# using this' every time I go to djangopackages.com. It becomes obvious that it is a misrepresentation of '# using it' when compared to github stars. 

Why does it not work? Because it is against DRY.,. People don't like to enter same information here and there.. 

So, I believe that field should be removed in favor of github stars which is already there. 

Regards,

**Comment by michaeljohnbarr** [comment]
I respectfully disagree. The star system was not designed to indicate use [according to the GitHub documentation](https://help.github.com/articles/about-stars/). I do agree that the number of GitHub stars might be an interesting metric to see.

**Comment by doganmeh** [comment]
Nevertheless, GitHub stars is an indicator of the popularity per the same
documentation linked, and it works, the other doesn't.

**Comment by sww314** [comment]
I agree the "using this" is not very useful. I think the pip downloads are good metric to check as well. That is one I typically use.

**Comment by ryancheley** [comment]
I think there are a couple of questions to consider with this:

1. Is this feature being used frequently enough to warrant update / refactor?
2. What is the methodology / process for administrator approval? 

If GitHub Stars are already (mostly) being used as a proxy, it may make more sense to just update the page to show the Number of Stars vs the 'Using it'
